docs/docs/users/guides/methods_filtering.md

```diff
@@ -104,7 +104,7 @@ If you are running a public RPC node, it is recommended to filter certain method
 # Creates a snapshot of the chain and writes it to a file. Very resource-intensive.
 !Filecoin.ChainExport
 # Potentially resource-intensive.
-!Filecoin.EthCall
-!eth_call
+!Filecoin.GasEstimateGasLimit
```

Comment:
Why this method? Why not `GasEstimateGasPremium`?

Reply:
The other method does not have parity test or snapshot yet

Reply:
Removed as this becomes fast with https://github.com/ChainSafe/forest/pull/5502